### PR TITLE
Added environment to templated SSHOperator fields

### PR DESCRIPTION
**Pull request by bdsoha**
Added `environment` as a templated field to the `SSHOperator`.

### CODE REVIEW

File: airflow/providers/ssh/operators/ssh.py [hunk 1]
```diff
@@ -61,9 +61,12 @@ class SSHOperator(BaseOperator):
     :param banner_timeout: timeout to wait for banner from the server in seconds
     """
 
-    template_fields: Sequence[str] = ('command', 'remote_host')
+    template_fields: Sequence[str] = ('command', 'environment', 'remote_host')
     template_ext: Sequence[str] = ('.sh',)
-    template_fields_renderers = {"command": "bash"}
+    template_fields_renderers = {
+        "command": "bash",
+        "environment": "json",
```
Comment (uranusjr) [inline airflow/providers/ssh/operators/ssh.py]: Since `environment` is a Python dict, shouldn’t this be `python` instead?

Comment (bdsoha) [reply airflow/providers/ssh/operators/ssh.py:68]: The values are later converted into strings *(as all environment variables)*.

Comment (bdsoha) [reply airflow/providers/ssh/operators/ssh.py]: @uranusjr Done, using `python` instead.

**Comment by bdsoha** [comment]
@uranusjr It seems like there was a transient failure on one of the tests.
Can you please re-trigger it?

**Review by uranusjr — APPROVED** [review]

**Review by o-nikolas — APPROVED** [review]

**Pull request merged**